### PR TITLE
fix: code visual changes

### PR DESCRIPTION
**Pull request by arunvjn**
## Description
This PR contains some UI changes for code in Appsmith.
- Sets font-family as ui-monospace for text in code editors and JSON viewer (used in evaluated value popup and debugger).
- Sets font-size of text in code editors to 13px.
- Set font-size for text in autocomplete to 11px.
- Removed border as the focus state for JS Editor.
- Added line highlight to indicate the active line. This will be the focus state of JS Editor going forward.
- Removed border-bottom for Tabs in JS Editor to keep it consistent with rest of the pages.
- Introduces new colors as mentioned in this [doc](https://www.notion.so/appsmith/Visual-changes-to-code-in-Appsmith-8bb530c60ee844e5b44adec039bf9280#d99e8450f022439f845b5a0d7deb1d3f).

#### TL;DR
Ever questioned why the JS Editor in Appsmith resembles a huge input box? Now you may put that worry to rest.

Fixes # (issue)
> if no issue exists, please create an issue and ask the maintainers about this first

#### Media
Screenshot 2023-01-16 at 12 24 02
Screenshot 2023-01-16 at 12 24 36
Screenshot 2023-01-16 at 12 25 53

#### Type of change
- Code editor upgrade

#### How Has This Been Tested?
- Manual

### Dev activity
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] PR is being merged under a feature flag

### QA activity:
- [ ] Test plan has been approved by relevant developers
- [ ] Test plan has been peer reviewed by QA
- [ ] Cypress test cases have been added and approved by either SDET or manual QA
- [ ] Organized project review call with relevant stakeholders after Round 1/2 of QA
- [ ] Added Test Plan Approved label after reveiwing all Cypress test

### CODE REVIEW

**Comment by ramsaptami** [comment]
Looks good at the moment - will review it again when Bharat's comments are incorporated.
CC: @arunvjn

**Review by bharath31 — CHANGES_REQUESTED** [review]
Shared my feedback [here](https://www.notion.so/appsmith/Bharath-s-thoughts-71840a5779b24d1f92ee0f68d5c9d009). Looking forward to @vinay-appsmith's updates on the designs.
